.github/workflows/reusable-prober.yml

```diff
@@ -203,79 +197,79 @@ jobs:
         run: |
           docker pull ghcr.io/linuxcontainers/alpine
           docker tag ghcr.io/linuxcontainers/alpine ${IMAGE}
-          docker push ${IMAGE}
+          docker push ${IMAGE}  
 
-      # START: PREPRODUCTION VERIFICATION:
-      #   * "Upgrade" cosign from prod to preprod TUF; sign and verify
-      #   * Do not run this test for staging TUF
-      # TODO: Create a matrix (https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs)
-      #       to reduce duplication
-
-      - name: Initialize prod TUF root
-        if: ${{ inputs.enable_staging == false }}
+      - name: Install tufie
         run: |
-          for i in {1..5}
-          do
-            if cosign initialize --mirror=${{ inputs.tuf_repo }} --root=${{ inputs.tuf_root_path }}; then
-              echo "Successfully initialized" && exit 0
-            else
-              echo "Failed to initialize" && sleep 10
-            fi
-          done
+          bash <(curl -s https://raw.githubusercontent.com/kairoaraujo/tufie/main/install.sh)
 
-      - name: Initialize preprod TUF root
-        if: ${{ inputs.enable_staging == false }}
+      - name: Set initial root.json
+        run: cp "${{ inputs.tuf_root_path }}" ./root.json
+
+      - name: Add TUF repository to tufie
         run: |
-          for i in {1..5}
-          do
-            if cosign initialize --mirror=${{ inputs.tuf_preprod_repo }} --root=${{ inputs.tuf_root_path }}; then
```

Comment:
So that we don't lose the signing and verification using the preprod repo as well, could we use a matrix that includes `tuf_preprod_repo` and `tuf_repo`?

Reply:
SG, I'm good with dropping preprod for simplicity. This made more sense when promoting preprod was a manual step and wasn't automated.